config/config_test.go

```diff
@@ -129,3 +131,83 @@ func TestParse(t *testing.T) {
 		}
 	}
 }
+
+// TestParseFields will test that fields are properly parsed from a config. As fields are added, then please
+// add tests here.
+func TestParseFields(t *testing.T) {
+	tomlConfig := `
+[[rules]]
+	description = "Some Groups without a reportGroup"
+	regex = '(.)(.)'
+
+[[rules]]
+	description = "Some Groups"
+	regex = '(.)(.)'
+  reportGroup = 1
+`
+	configPath, err := writeTestConfig(tomlConfig)
```

Comment:
**suggestion**: @squaresurf What about loading the config from a file in `test_data/test_configs`?